obsreport/obsreport_scraper.go

```diff
@@ -118,3 +176,15 @@ func (s *Scraper) EndMetricsOp(
 
 	span.End()
 }
+
+func (s *Scraper) recordMetrics(scraperCtx context.Context, numScrapedMetrics, numErroredMetrics int) {
+	if s.useOtelForMetrics {
+		s.scrapedMetricsPoints.Add(scraperCtx, int64(numScrapedMetrics), s.otelAttrs...)
+		s.erroredMetricsPoints.Add(scraperCtx, int64(numErroredMetrics), s.otelAttrs...)
+	} else { // OC for metrics
+		stats.Record(
+			scraperCtx,
+			obsmetrics.ScraperScrapedMetricPoints.M(int64(numScrapedMetrics)),
+			obsmetrics.ScraperErroredMetricPoints.M(int64(numErroredMetrics)))
+	}
+}
```

Comment:
```suggestion
}

```

Reply:
Weird I thought I added this newline. Something in git must be preventing me from adding a newline at the end of the file (linter?)